internal/pkg/deploy/cloudformation/errors.go

```diff
@@ -28,3 +28,18 @@ type ErrNotExecutableChangeSet struct {
 func (err *ErrNotExecutableChangeSet) Error() string {
 	return fmt.Sprintf("cannot execute change set %s because status is %s with reason %s", err.set, err.set.executionStatus, err.set.statusReason)
 }
+
+// ErrTemplateNotFound occurs when we can't find a predefined template.
+type ErrTemplateNotFound struct {
```

Comment:
nit: I learned recently that if you add a `Is` method to an error struct, you can leverage `errors.Is` even though this is not a sentinel error!
see: [Customizing error tests with Is and As methods](https://blog.golang.org/go1.13-errors#TOC_4.)